### PR TITLE
Describe Franklin subscription filter

### PR DESCRIPTION
**Pull request by KmilleF**

### CODE REVIEW

File: content/md/franklin-insights/franklin-insights-subscriptions.md [hunk 1]
```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
...
-- Via **bulk action**
+## Via **bulk action**
 
 ![image](../img/Akeneo_Screenshot_Franklin_Insights_Bulk_Actions.png)
 
 In order for you to receive attribute values proposals for subscribed and enriched products, you need to **match between Franklin’s attributes and your catalog attributes** in each family using the [Franklin attributes mapping UI](franklin-insights-mappings.html#identifiers-mapping).
 
+# Filter on subscribed products in the product grid (_since 3.1_)
+
+In order to easily find the subscribed or unsubscribed products, we added a filter `Franklin subscription` in the product grid. Thus, you can easily display the products in the grid and [bulk subscribe or unsubscribe] them.
```
Comment (NolwennP) [inline content/md/franklin-insights/franklin-insights-subscriptions.md]: Why the brackets? "[bulk subscribe or unsubscribe]"

Comment (KmilleF) [reply content/md/franklin-insights/franklin-insights-subscriptions.md]: I wanted to add a link to an article and forgot it. Fix in progress

File: content/md/franklin-insights/franklin-insights-subscriptions.md [hunk 2]
```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
 
 You have 2 ways to subscribe products to Franklin:
 
-- Via the **Product Edit Form**
+## Via the **Product Edit Form**
 
 ![image](../img/Akeneo_Screenshot_Franklin_Insights_PEF_Subscription.png)
 
-- Via **bulk action**
+## Via **bulk action**
```
Comment (NolwennP) [inline content/md/franklin-insights/franklin-insights-subscriptions.md]:
```suggestion
## Via bulk action
```

File: content/md/franklin-insights/franklin-insights-subscriptions.md [hunk 3]
```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
 
 You have 2 ways to subscribe products to Franklin:
 
-- Via the **Product Edit Form**
+## Via the **Product Edit Form**
```
Comment (NolwennP) [inline content/md/franklin-insights/franklin-insights-subscriptions.md]:
```suggestion
## Via the Product Edit Form
```

File: content/md/franklin-insights/franklin-insights-subscriptions.md [hunk 4]
```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
 
 You have 2 ways to subscribe products to Franklin:
```
Comment (NolwennP) [inline content/md/franklin-insights/franklin-insights-subscriptions.md]:
```suggestion
You have 2 ways to subscribe products to Franklin.
```

File: content/md/franklin-insights/franklin-insights-subscriptions.md (hunk 1 above)
Comment (NolwennP) [reply content/md/franklin-insights/franklin-insights-subscriptions.md]:
```suggestion
In order to easily find the subscribed or unsubscribed products, we added a filter `Franklin subscription` in the product grid. Thus, you can easily display the products in the grid and [bulk subscribe or unsubscribe](franklin-insights-subscriptions.html#via-bulk-action) them.
```

**Pull request merged**